src/main/java/homework_2/RandomCharsTable.java

```diff
@@ -0,0 +1,85 @@
+package homework_2;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Random;
+
+public class RandomCharsTable {
+
+    private static int arg1;
+    private static int arg2;
+    private static String arg3;
+    private static boolean even;
+
+    public static void main(String[] args) {
+
+        try (BufferedReader reader = new BufferedReader(new InputStreamReader(System.in))) {
+
+            String s = reader.readLine();
+
+            if (chekOk(s)) {
+
+                char[][] arr = new char[arg1][arg2];
+                Random r = new Random();
+                StringBuilder b = new StringBuilder();
+                StringBuilder log = new StringBuilder();
+
+                for (char[] c1 : arr) {
+                    for (char c2 : c1) {
+                        int num = 65 + r.nextInt(26);
+                        c2 = (char) (num);
+                        if (num % 2 == 0 && even) {
+                            b.append(c2);
```

Comment:
Opt: Even/odd letters are concatenated together "GPSDP". Please add whitespace and comma between them